package.json

```diff
@@ -79,8 +87,11 @@
     "gatsby-source-filesystem": "^4.6.0",
     "gatsby-transformer-sharp": "^4.6.0",
     "http-proxy-middleware": "^1.3.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^28.1.1",
     "prettier": "2.5.1",
     "react-is": "^17.0.2",
+    "react-test-renderer": "^18.2.0",
```

Comment:
I think initially jest should only unit test services and have cypress handle mocked end to end. Setting up component unit testing should be most likely deferred (it's a bit much, we can get back to that once we tackled basic testing everywhere else). What do you think @meeh0w ?

Reply:
Fair enough 👍